### PR TITLE
Support for generating QR which includes ByteData

### PR DESCRIPTION
**Pull request by Yamaguchi**
This PR will solve issue #25

### CODE REVIEW

**Comment by Yamaguchi** [comment]
Example code on flutter is the following:

```
import 'package:flutter/material.dart';
import "dart:typed_data";
import 'package:qr/qr.dart';
import 'package:qr_flutter/qr_flutter.dart';

class QrPage extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text("QR"),
      ),
      body: _buildQR(),
    );
  }

  _buildQR() {
    var data = new ByteData(256);
    for (var i = 0; i < 256; i++) {
      data.setUint8(i, i);
    }

    final qrCode = new QrCode(14, QrErrorCorrectLevel.Q);
    qrCode.addByteData(data);
    qrCode.make();

    return QrImage.withQr(qr: qrCode);
  }
}
```
(https://gist.github.com/Yamaguchi/4e56f0e448de06fb3a830abbc62baac6)

**Comment by Yamaguchi** [comment]
The above code generate QR image like this:

qr

File: lib/src/qr_code.dart [hunk 1]
```diff
@@ -78,6 +79,12 @@ class QrCode {
     _dataCache = null;
   }
 
+  void addByteData(ByteData data) {
```
Comment (kevmoo) [inline lib/src/qr_code.dart]: Would you create a private helper that can be called by both `addData` and `addByteData`?

File: lib/src/byte.dart [hunk 2]
```diff
@@ -13,6 +14,10 @@ class QrByte {
     return QrByte._internal(charUnits);
   }
 
+  factory QrByte.fromByteData(ByteData input) {
+    return QrByte._internal(input.buffer.asUint8List().toList());
```
Comment (kevmoo) [inline lib/src/byte.dart]: Use `=>` syntax here!

**Comment by Yamaguchi** [comment]
Thank you for reviewing. I added commits(, and rebase from master).

File: pubspec.yaml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 name: qr
-version: 1.2.1-dev
+version: 1.2.2-dev
```
Comment (kevmoo) [inline pubspec.yaml]: since this is a new feature, it should be `1.3.0-dev`

Comment (Yamaguchi) [reply pubspec.yaml]: I'll fixed it.

**Pull request merged**